api.bs

```diff
@@ -1414,6 +1414,9 @@ To perform <dfn>common matching logic</dfn>, given
 1.  [=set/iterate|For each=] |impression| in the [=impression store=] for the |epoch|:
 <!-- TODO: Clarify "for the |epoch|" -->
 
+    1.  If |now| - |impression|'s [=impression/lifetime=] is after |impression|'s
+        [=impression/timestamp=], [=iteration/continue=].
```

Comment:
I generally shape time comparison like this:

```suggestion
    1.  If |now| is after |impression|'s [=impression/timestamp=] plus |impression|'s
        [=impression/lifetime=], [=iteration/continue=].
```

(I've learned that some time systems don't let you go backwards past certain points reliably.)